core/kmod/llring.h

```diff
@@ -97,7 +97,11 @@
 #define _LLRING_H_
 
 #if !defined(__cplusplus) // C
+#ifdef fallthrough
```

Comment:
shouldn't this be?

```suggestion
#ifndef fallthrough
```

Reply:
Thanks for the suggestion! `#ifdef` is intended. This is to avoid collision with `fallthrough` from torvalds/linux@294f69e. Preprocessor tries to replace `fallthrough` inside `__attribute__(())` if it is already defined. I guess there may be better ways for resolving this issue...

Reply:
oh doh! we're defining `FALLTHROUGH` not `fallthrough`-- my bad. Thanks for clarifying. This is OK as-is